synapse/config/room.py

```diff
@@ -50,7 +50,12 @@ def read_config(self, config, **kwargs):
                 RoomCreationPreset.PRIVATE_CHAT,
                 RoomCreationPreset.TRUSTED_PRIVATE_CHAT,
             ]
-        elif encryption_for_room_type == RoomDefaultEncryptionTypes.OFF:
+        elif (
+            encryption_for_room_type == RoomDefaultEncryptionTypes.OFF
+            or not encryption_for_room_type
```

Comment:
Would an `is False` check be better here? Otherwise other falsy values (like None or empty string) will match.

Reply:
Good shout, especially as we generally try to go in that direction. Fixed in https://github.com/matrix-org/synapse/pull/7822/commits/68a25ed14c86add34ac0db46e1461b381b953a5f